addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -43,7 +43,8 @@ <H2>Application Errors</H2>
 Check server responses for HTTP 500 - Internal Server Error type responses or those that contain a known error string. <br>
 <strong>Note:</strong> Matches made within script blocks or files are against the entire content not only comments.<br>
 At HIGH Threshold don’t alert on HTTP 500 (but do for other error patterns).<br>
-For Internal Server Error (HTTP 500) the Alert is set to Low risk and in other case it is set to Medium risk.
+For Internal Server Error (HTTP 500) the Alert is set to Low risk and in other case it is set to Medium risk.<br>
+Errors like this are much less likely in static pages like JS / CSS so they are only raised at a LOW threshold in case of these type of responses.
```

Comment:
Move the sentence to immediately after the other one that talks about the threshold. Also, "Errors like this" could be explicit to avoid confusions between the two errors being checked.